nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -91,6 +95,7 @@ blocks:
     ops:
       # aka insertOne
       write-document:
+        space: "{space}"
```

Comment:
Do we have to add this to every instruction because of a shortcoming in NoSQLBench?

Reply:
Yes, see: https://github.com/nosqlbench/nosqlbench/issues/1182